### PR TITLE
Update isort, Python, and PostgreSQL.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -8,7 +8,7 @@ passenv = DJANGO_SETTINGS_MODULE
 deps =
     tests: -r{toxinidir}/requirements/tests.txt
     flake8: flake8
-    isort: isort
+    isort: isort >= 5.1.0
```
Comment (carltongibson) [inline tox.ini:11]: I don't see the need to specify the `>` here. tox will install the latest when building the env. That'll always be `>` some old version.

Comment (felixxm) [reply tox.ini:11]: I added it because `isort < 5` requires `recursive` flag, so we should upgrade local env before running `tox`.

Comment (carltongibson) [reply tox.ini:11]: > ...so we should upgrade local env before...

Do you mean when running ourselves, _locally_? In which case I'd say _meh_ — just `--recreate` if tox didn't do that itself. 

Or do we need this to force the CI environment to rebuild? (Does it not do that every run?)

Comment (felixxm) [reply tox.ini]: Removed. You're right tox recreates env on every run.

**Pull request merged**